docs/contrib/mangling.rst

```diff
@@ -54,4 +54,4 @@ that uses a notation inspired by
         j                              // scala.Long
 
     <name> ::=
-        <length number> <chars>        // raw identifier of given length
+        <length number> [-] <chars>    // raw identifier of given length; `-` separator is only used when <chars> starts with digit or `-` itself
```

Comment:
This change was already merged in another PR. If you rebase on master it should disappear from here.

Reply:
Gone now.  An intended http to https change remains.